spec/features/visitor/sign_up_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+feature "Sign Up" do
+  let(:user_attributes) { attributes_for(:user).slice(:full_name, :email, :password, :password_confirmation) }
+  let(:registered_user) { User.find_by_email(user_attributes[:email]) }
+
+  background do
+    visit new_user_registration_path
+
+    fill_form(:user, user_attributes)
+
+    click_button "Sign up"
+  end
+
+  scenario "Visitor signs up" do
+    open_email(registered_user.email)
+
+    expect(current_email).to have_subject "Confirmation instructions"
+    expect(current_email).to have_body_text(registered_user.full_name)
+  end
+
+  scenario "Visitor confirms account" do
```

Comment:
@vast what do you think if we will extract this scenario as a separate feature "Confirm Account" since we have separate feature "Resend Confirmation Email"?